scbs/MethylVI_batch.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p>Maybe show get_normalized_methylation or DE additionally to go beyond latent space analysis? See our other tutorials for examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Once this notebook is merged in I plan to create another tutorial for feature-level analysis</p><p> </p>

Reply:
<p>Can we have it in one notebook? I don't like bloating the number of tutorials. It's currently already quite painful to rerun and manually merge notebooks. Do you need different datasets for it?</p>

Reply:
For feature-level analysis I would prefer to use a different dataset (I don't have as good ground truths for the batch effect dataset). I'm happy to put both analyses in a single notebook though if that makes things easier for you?

Reply:
No if it's using different datasets, make it different notebooks.